addons/ekco/template/base/reboot/shutdown.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 export KUBECONFIG=/etc/kubernetes/kubelet.conf
+KURL_INSTALL_DIRECTORY=/var/lib/kurl
```

Comment:
this is technically configurable

see https://kurl.sh/docs/install-with-kurl/advanced-options `kurl-install-directory`

Reply:
The only way to ensure that is if we in the install script overwrite this value. 
Note that those scripts are shipped with the add on.